snakes_battle/board.py

```diff
@@ -57,3 +59,12 @@ def get_board_state(self):
             "snakes": copy.deepcopy(self.snakes),
             "fruits": copy.deepcopy(self.fruits)
             }
+
+    def total_game_time(self):
+        return time.time() - self.start_time
+    
+    def game_time_left(self):
+        return round(settings.GAME_TIME_LENGTH - self.total_game_time(), 2)
+    
+    def is_game_timed_out(self):
+        return self.game_time_left() <= 0
```

Comment:
In my opinion, this code block should be in main module. The timer should start when entering the loop in 'run_game' function.